app/templates/editions/octane.hbs

```diff
@@ -39,7 +42,7 @@
   </dd>
   <dt>I'm starting a new app at work. Should I use Octane?</dt>
   <dd>
-    Octane is in <em>preview</em> mode- there are bound to be rough edges, and upcoming changes. Since we don't know <i>what</i> we don't know just yet, we advise that Octane be used for experimentation only. We recommend that you continue to use stable versions of Ember for new production applications. 
+    Octane is in <em>preview</em> mode- there are bound to be rough edges in popular addons and tools. We advise that Octane style only be used in v3.13 apps by developers who are committed to helping iron out any issues. Developers are encouraged to update their apps to the latest version of Ember in the same way they always would. v3.13 has the stability guarantees that minor releases always have, and is production-ready. Octane's opt-in features will get more ecosystem support and documentation built up in the meantime.
```

Comment:
v3.13 is production-ready and has the stability guarantees that minor releases always have.

I'd flip these two clauses- just a suggestion though.